### PR TITLE
Update elements - Closes #448

### LINKED ISSUE #448: Update Lisk Elements package reference
* include new packages
* update existing packages reference to support lisk-sdk 3.x

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #448

### CODE REVIEW

File: modules/ROOT/pages/reference/lisk-elements/packages/cryptography.adoc [hunk 1]
```diff
@@ -154,6 +154,35 @@ const hex = 'abcd1234';
...
+
+[source,js]
+----
+intToBuffer(intString, byteLength, endianness)
+----
+
+==== Parameters
+
+* `intString`: The integer to convert to a buffer. Can be of type `string` or `number`
+* `byteLength`: The integer to convert to a buffer.
```
Comment (michielmulders) [inline modules/ROOT/pages/reference/lisk-elements/packages/cryptography.adoc]: Is this line 171 correct?

File: modules/ROOT/pages/reference/lisk-elements/packages/cryptography.adoc [hunk 2]
```diff
@@ -219,6 +248,33 @@ const encryptedPassphrase = cryptography.encryptPassphraseWithPassword(
...
+==== Syntax
+
+[source,js]
+----
+stringToBuffer(string)
+----
+
+==== Parameters
+
+`string`: A string value.
```
Comment (michielmulders) [inline modules/ROOT/pages/reference/lisk-elements/packages/cryptography.adoc:266]: This is very dry information? 😃

File: modules/ROOT/pages/reference/lisk-elements/packages/cryptography.adoc [hunk 3]
```diff
@@ -344,12 +400,12 @@ This encrypts a passphrase under a password for secure storage.
 
 [source,js]
 ----
-encryptPassphraseWithPassword(passphrase, password, [iterations])
+encryptPassphraseWithPassword(plainText, password, [iterations])
```
Comment (michielmulders) [inline modules/ROOT/pages/reference/lisk-elements/packages/cryptography.adoc]: I'm fine with plainText, but as the function is called `encryptPassphrase...` does it make more sense to keep `passphrase` or you prefer `plainText`?

File: modules/ROOT/pages/reference/lisk-elements/packages/cryptography.adoc (hunk 2 above)
Comment (michielmulders) [reply modules/ROOT/pages/reference/lisk-elements/packages/cryptography.adoc:266]: But yea, it's in the end string to buffer, so what can we say! :)

File: modules/ROOT/pages/reference/lisk-elements/packages/p2p.adoc [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+It will start a node that is available on `5001` websocket port.
+
+[source,bash]
+----
+P2P node has started!
+----
+
+=== Actions
+
+It provides simple interface to send, request, broadcast information and many more functions to interact with the network.
```
Comment (michielmulders) [inline modules/ROOT/pages/reference/lisk-elements/packages/p2p.adoc:62]: Very useful info!

**Review by michielmulders — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/reference/lisk-elements/packages/cryptography.adoc (hunk 1 above)
Comment (Tschakki) [reply modules/ROOT/pages/reference/lisk-elements/packages/cryptography.adoc]: nope ^^ will fix

**Review by michielmulders — APPROVED** [review]
Cool to go! :)

**Pull request merged**